open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring/src/main/resources/application.properties

```diff
@@ -6,20 +6,21 @@ server.compression.enabled=true
 server.compression.mime-types=application/json,application/xml,text/html,text/xml,text/plain,application/javascript,text/css
 server.tomcat.uri-encoding=UTF-8
 
-server.ssl.key-store: classpath:keystore.p12
-server.ssl.key-store-password: 123456
-server.ssl.keyStoreType: PKCS12
-server.ssl.keyAlias: tomcat
+server.ssl.key-store=classpath:keystore.p12
+server.ssl.key-store-password=egeria
+server.ssl.keyStoreType=PKCS12
+server.ssl.keyAlias=tomcat
 
-# set to false if you want to ignore invalid ssl certificate of a server you want to connect to
+# WARNING! setting 'false' allows java clients to open https connections without checking certificates validity
+# Alternate you can import self signed certificates into java truststore or setup an truststore only for this app
+# by adding the store into server.ssl.trust-store parameter
 strict.ssl=true
```

Comment:
will it work OOTB with strict.ssl=true? If not how do we ensure it's obvious/simple/clear to get up (less of an issue in the container env where we can easily setup the certs - or disable for tutorial (less preferable))